account_cash_invoice/wizard/cash_invoice_in.xml

```diff
@@ -8,11 +8,15 @@
             <form string="Pay supplier invoice or customer refund">
                 <group>
                     <field name="company_id" invisible="1"/>
-                    <field name="journal_ids" invisible="1"/>
+                    <field name="journal_ids" invisible="1">
+                        <tree>
+                            <field name="id"/>
```

Comment:
Is this really needed?

Reply:
On the cash_invoice_id view is probably not needed but at least if there is more than 1 journal to select, it will not crash an error.